guess_the_words/function.py

```diff
@@ -0,0 +1,161 @@
+import json
+import string
+import random
+import datetime
+
+
+def create_list():
+    with open('words.txt', 'r') as file:
+        list_words = file.readlines()
+        for posit, word in enumerate(list_words):
+            list_words[posit] = word.strip()
+    return list_words
+
+
+def val_count(player_letters: list) -> list:
+    if len(player_letters) > 5:
+        print('You enter word more then 5 letters')
+        return True
+    elif len(player_letters) < 5:
+        print('You enter word less then 5 letters')
+        return True
+
+
+def rules_for_user():
+    with open('Rules.txt', 'r') as file:
+        open_file = file.read()
+    return open_file
+
+
+def hello_choose(player_name: str) -> str:
+    choose_eser = input(f'Hello, {player_name}. Choose what you need:'
+                        f'\n1. Read rules of game - type "rules"'
+                        f'\n2. To start game - type "start"'
+                        f'\n3. To stop game - type "stop"'
+                        f'\n4. Show past result - type "result"')
+    return choose_eser
+
+
+def search_result(name_player: str) -> str:
+    with open('guess_result.json', 'r') as file:
+        data = json.load(file)
+        list_data = data['Result of games']
+        ii = 0
+        while ii < len(list_data):
+            list_j = []
+            ii += 1
+            for i, j in enumerate(list_data):
+                if name_player == j['Player name']:
+                    list_j.append(j)
+            return list_j
+
+
+def choose_loop(choose_user: str, player_name: str) -> str:
+    while True:
+        if choose_user == 'rules'.lower():
+            rules = rules_for_user()
+            print(rules)
+            break
+        elif choose_user == 'stop'.lower():
+            print('Game was stopped')
+            break
+        elif choose_user == 'start'.lower():
+            body_loop(player_name)
+            break
+        elif choose_user == 'result'.lower():
+            name_player = input('Enter player name: ').title()
+            result_search = search_result(name_player)
+            if result_search == []:
```

Comment:
Just the better practice:
```suggestion
            if not len(result_search):
```

Reply:
And much better to check:
```
if len(result_search):
    ...
else:
    ...
```

Reply:
Thank you. I saw what result function was and set conditions result_search == []  ))) Corrected.